app/com/arpnetworking/metrics/portal/reports/Report.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2018 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.metrics.portal.reports;
+
+import com.google.common.base.MoreObjects;
+
+import java.util.Arrays;
+import java.util.Objects;
+import javax.annotation.Nullable;
+
+/**
+ * Snapshot of a fully rendered report.
+ *
+ * @author Spencer Pearson
+ */
+public final class Report {
+
+    /**
+     * @param html The HTML version of the report, if that makes sense (else null).
```

Comment:
Missing method description; e.g. `Public constructor.`

Reply:
Refactored this to use the Builder pattern instead.